requirements.txt

```diff
@@ -1,3 +1,3 @@
-pennylane>=0.15
+git+https://github.com/PennyLaneAI/pennylane.git
```

Comment:
I also updated this to point to PL master, since `qml.Projector` is not present in the stable v0.15 release